packages/view/backend/sirius-components-view-form/src/main/resources/model/form.ecore

```diff
@@ -80,6 +80,10 @@
         eType="ecore:EDataType ../../../../../sirius-components-view/src/main/resources/model/view.ecore#//Length"
         defaultValueLiteral="250"/>
   </eClassifiers>
+  <eClassifiers xsi:type="ecore:EClass" name="SplitButtonDescription" eSuperTypes="#//WidgetDescription">
```

Comment:
Why there is no "isEnabledExpression" on this widget?

Reply:
The button is disabled is the current action is disabled

Reply:
Yes but it also means that the "dropdown button" will never be disabled. Do we really want that? May be, may be not, please be sure about that.